code/game/objects/items/weapons/chaplain_items.dm

```diff
@@ -193,3 +193,58 @@
 			A.dropInto(loc)
 			user.visible_message("[user] pours \the [A] out from \the [src].", "You pour \the [A] out from \the [src].")
 			desc = "A vase used to store the ashes of the deceased."
+
+/obj/item/assunzioneorb
+	name = "warding sphere"
+	desc = "A religious artefact commonly associated with Luceism, this transparent globe gives off a faint ghostly white light at all times."
+	desc_fluff = "Luceian warding spheres are made on the planet of Assunzione in the great domed city of Guelma, and are carried by followers of the faith heading abroad. \
+	Constructed out of glass and a luce vine bulb these spheres can burn for years upon years, and it is said that the lights in the truly faithful's warding sphere will always \
+	point towards Assunzione. It is considered extremely bad luck to have one's warding sphere break, to extinguish its flame, or to relinquish it (permanently) to an unbeliever."
+	icon = 'icons/obj/weapons.dmi'
+	icon_state = "assunzioneorb"
+	item_state = "assunzioneorb"
+	light_range = 1.4
+	light_power = 1.4
+	light_color = LIGHT_COLOR_BLUE
+	w_class = ITEMSIZE_SMALL
+	drop_sound = 'sound/items/drop/glass.ogg'
+	pickup_sound = 'sound/items/pickup/glass.ogg'
+
+/obj/item/assunzioneorb/proc/shatter()
+	visible_message(SPAN_WARNING("\The [src] shatters!"), SPAN_WARNING("You hear a small glass object shatter!"))
+	force = 5
+	sharp = TRUE
```

Comment:
What is the purpose of doing this if the item will be deleted?

Reply:
It's there so that there's a prompt that says it broke, similar to other glass items.

Reply:
This var is pointless, remove it.

Reply:
force and sharp have been removed.
I misunderstood the code and thought sharp made it cause bleeding. As for the force, I transferred it to the item itself.